x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/email/email.tsx

```diff
@@ -75,6 +75,26 @@ export function getActionType(): ActionTypeModel<EmailActionConnector, EmailActi
           )
         );
       }
+      if (action.config.hasAuth && !action.secrets.user && !action.secrets.password) {
```

Comment:
Should this check username only?

```suggestion
      if (action.config.hasAuth && !action.secrets.user) {
```